From ee926ea1bd602095ba6d5f148451c93e9712beb9 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Fri, 16 Aug 2019 00:31:02 +0200
Subject: [PATCH 57/60] syslogd: Add a syslog.conf file

Add a default configuration file so that the users have something to
start from.

Ref: https://bugs.debian.org/208336
---
 src/Makefile.am |  2 ++
 src/syslog.conf | 70 +++++++++++++++++++++++++++++++++++++++++++++++++
 2 files changed, 72 insertions(+)
 create mode 100644 src/syslog.conf

diff --git a/src/Makefile.am b/src/Makefile.am
index 7682d713..eef2254a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -107,6 +107,8 @@ inetdaemon_PROGRAMS += $(syslogd_BUILD)
 syslogd_SOURCES = syslogd.c
 EXTRA_PROGRAMS += syslogd
 
+dist_sysconf_DATA = syslog.conf
+
 inetdaemon_PROGRAMS += $(tftpd_BUILD)
 tftpd_SOURCES = tftpd.c
 EXTRA_PROGRAMS += tftpd
diff --git a/src/syslog.conf b/src/syslog.conf
new file mode 100644
index 00000000..5b35eb8c
--- /dev/null
+++ b/src/syslog.conf
@@ -0,0 +1,70 @@
+#  /etc/syslog.conf	Configuration file for inetutils-syslogd.
+#
+#			For more information see syslog.conf(5) manpage.
+
+#
+# First some standard logfiles.  Log by facility.
+#
+
+auth,authpriv.*			/var/log/auth.log
+*.*;auth,authpriv.none		-/var/log/syslog
+#cron.*				/var/log/cron.log
+daemon.*			-/var/log/daemon.log
+kern.*				-/var/log/kern.log
+lpr.*				-/var/log/lpr.log
+mail.*				-/var/log/mail.log
+user.*				-/var/log/user.log
+uucp.*				/var/log/uucp.log
+
+#
+# Logging for the mail system.  Split it up so that
+# it is easy to write scripts to parse these files.
+#
+mail.info			-/var/log/mail.info
+mail.warn			-/var/log/mail.warn
+mail.err			/var/log/mail.err
+
+# Logging for INN news system
+#
+news.crit			/var/log/news/news.crit
+news.err			/var/log/news/news.err
+news.notice			-/var/log/news/news.notice
+
+#
+# Some `catch-all' logfiles.
+#
+*.=debug;\
+	auth,authpriv.none;\
+	news.none;mail.none	-/var/log/debug
+*.=info;*.=notice;*.=warn;\
+	auth,authpriv.none;\
+	cron,daemon.none;\
+	mail,news.none		-/var/log/messages
+
+#
+# Emergencies are sent to everybody logged in.
+#
+*.emerg				*
+
+#
+# I like to have messages displayed on the console, but only on a virtual
+# console I usually leave idle.
+#
+#daemon,mail.*;\
+#	news.=crit;news.=err;news.=notice;\
+#	*.=debug;*.=info;\
+#	*.=notice;*.=warn	/dev/tty8
+
+# The named pipe /dev/xconsole is for the `xconsole' utility.  To use it,
+# you must invoke `xconsole' with the `-file' option:
+#
+#    $ xconsole -file /dev/xconsole [...]
+#
+# NOTE: adjust the list below, or you'll go crazy if you have a reasonably
+#      busy site..
+#
+daemon.*;mail.*;\
+	news.crit;news.err;news.notice;\
+	*.=debug;*.=info;\
+	*.=notice;*.=warn	|/dev/xconsole
+
-- 
2.26.0.292.g33ef6b2f38

